glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: d15c7d65c510a024e73ba72a84be2c57d237cef2ab03d7c87af9ff302284645d
-updated: 2019-09-09T15:25:08.380025912-04:00
+updated: 2020-02-21T15:36:28.140353-05:00
```

Comment:
ah glide .... I remember the days when I preferred it to go mod :-)

Reply:
yup - once upstream added go.mod compatible tags and got past the growing pains, I don't want to go back.

I have a separate backport PR to 4.2 that is awful right now. Need to pin SHAs everywhere.